packages/react-native-bpk-component-chip/src/BpkChipWrapper.js

```diff
@@ -101,59 +126,74 @@ const BpkChipWrapper = (props: Props) => {
     label,
     selected,
     style,
+    type,
     theme,
     ...rest
   } = props;
 
+  const userStyle = [style];
   const innerStyle = [styles.inner];
   const textStyle = [styles.text];
   const iconStyle = [styles.icon];
 
+  if (type === CHIP_TYPES.outline) {
+    userStyle.push({ elevation: 0 });
```

Comment:
This is a bit 'magic' at the moment, it's not immediately obvious why this is being added. How about adding a new style called `outerOutline` and then applying it here?

Reply:
This is due to the touchable overlay naming convention here which was created due to a bug in android but happy to change it and push() into userStyle